src/components/UnovaSVG.html

```diff
@@ -0,0 +1,705 @@
+        , width: 7, height: 3
+        , x: 63, y: 13
+        })
+    %>
+
+    <!--Route 13-->
+    <%- include('templates/mapRoute',
+        { route: 13, region: 4
+        , width: 7, height: 3
+        , x: 74, y: 13
+        })
+    %>
+    <!--Extra bit of Route 13-->
+    <%- include('templates/mapRoute',
+        { route: 13, region: 4
+        , width: 3, height: 1
+        , x: 78, y: 16
+        , noText: true
+        })
+    %>
+    <!--Extra bit of Route 13-->
+    <%- include('templates/mapRoute',
+        { route: 13, region: 4
+        , width: 9, height: 3
+        , x: 78, y: 17
+        , noText: true
+        })
+    %>
+
+    <!--Route 14-->
+    <%- include('templates/mapRoute',
+        { route: 14, region: 4
+        , width: 13, height: 3
+        , x: 74, y: 25
+        })
+    %>
+    <!--Extra bit of Route 14-->
+    <%- include('templates/mapRoute',
+        { route: 14, region: 4
+        , width: 3, height: 1
+        , x: 84, y: 24
+        , noText: true
+        })
+    %>
+    <!--Extra bit of Route 14-->
+    <%- include('templates/mapRoute',
+        { route: 14, region: 4
+        , width: 3, height: 3
+        , x: 74, y: 28
+        , noText: true
+        })
+    %>
+
+    <!--Route 15-->
+    <%- include('templates/mapRoute',
+        { route: 15, region: 4
+        , width: 6, height: 3
+        , x: 62, y: 32
+        })
+    %>
+
+    <!--Route 16-->
+    <%- include('templates/mapRoute',
+        { route: 16, region: 4
+        , width: 6, height: 3
+        , x: 56, y: 32
+        })
+     %>
+
+    <!--Route 17-->
+    <%- include('templates/mapRoute',
+        { route: 17, region: 4
+        , width: 15, height: 3
+        , x: 70, y: 55
+        })
+    %>
+     <!--Extra bit of Route 17-->
+    <%- include('templates/mapRoute',
+        { route: 17, region: 4
+        , width: 5, height: 3
+        , x: 85, y: 52
+        , noText: true
+        })
+    %>
+
+    <!--Route 18-->
+    <%- include('templates/mapRoute',
+        { route: 18, region: 4
+        , width: 8, height: 3
+        , x: 70, y: 52
+        })
+     %>
+
+    <!--Route 19-->
+    <%- include('templates/mapRoute',
+        { route: 19, region: 4
+        , width: 9, height: 3
+        , x: 7, y: 43
+        , rotate: true
+        })
+     %>
+     <!--Extra bit of Route 19-->
+     <%- include('templates/mapRoute',
+        { route: 19, region: 4
+        , width: 1, height: 3
+        , x: 10, y: 43
+        , noText: true
+        })
+    %>
+
+    <!--Route 20-->
+    <%- include('templates/mapRoute',
+        { route: 20, region: 4
+        , width:13, height: 3
+        , x: 15, y: 43
+        })
+    %>
+
+    <!--Route 21-->
+    <%- include('templates/mapRoute',
+        { route: 21, region: 4
+        , width: 6, height: 3
+        , x: 91, y: 10
+        , rotate: true
+        })
+    %>
+
+    <!--Undella Bay-->
+    <%- include('templates/mapRoute',
+        { route: Undella Bay, region: 4
```

Comment:
The map crashes right now because this is an invalid object
```suggestion
        { route: "Undella Bay", region: 4
```